source/proof_address_mock/lib/proof_address_mock.rb

```diff
@@ -0,0 +1,69 @@
+require 'proofer'
+require 'faraday'
+require 'retries'
+require 'identity-idp-functions/mock_proofers/address_mock'
+
+module IdentityIdpFunctions
+  class ProofAddressMock
+    def self.handle(event:, context:, &callback_block)
+      params = JSON.parse(event['body'], symbolize_names: true)
+      new(
+        idp_api_auth_token: ENV['IDP_API_AUTH_TOKEN'],
```

Comment:
let's make this `ENV.fetch` so we error if it's missing a value

Reply:
I made it optional because it doesn't get set in development/test environments.

We could set it in those environments, but we'd need a different name for the keys since there are different auth tokens for the address and resolution API endpoints.

Reply:
Oh good point!